unique_toolkit/unique_toolkit/app/unique_settings.py

```diff
@@ -19,9 +39,43 @@ class UniqueApp(BaseSettings):
     )
 
 
-class UniqueAuth(BaseSettings):
-    company_id: SecretStr
-    user_id: SecretStr
+class UniqueApi(BaseSettingsWithWarnings):
+    base_url: str = Field(default="http://localhost:8092/")
+    version: str = Field(default="2023-12-06")
+
+    model_config = SettingsConfigDict(
+        env_prefix="unique_api_",
+        env_file_encoding="utf-8",
+        case_sensitive=False,
+        extra="ignore",
+    )
+
+    def sse_url(self, subscriptions: list[str]) -> str:
+        parsed = urlparse(self.base_url)
+        return urlunparse(
+            parsed._replace(
+                path="/public/event-socket/events/stream",
+                query=f"subscriptions={','.join(subscriptions)}",
+                fragment=None,
+            )
+        )
+
+    def sdk_url(self) -> str:
+        parsed = urlparse(self.base_url)
+        return urlunparse(
+            parsed._replace(path="/public/chat", query=None, fragment=None)
```

Comment:
This might be misleading for next. (Works fine for all other deployments i believe)